openpgp/add.go

```diff
@@ -405,9 +371,9 @@ UPDATE openpgp_pubkey SET revsig_uuid = $1 WHERE uuid = $2`,
 
 func (w *Worker) updateSubkeyRevsig(tx *sqlx.Tx, subkey *Subkey, r *Signature) error {
```

Comment:
Ah, that's not actually true. But I think is still the correct change to introduce (given the desire to make this and other methods agnostic of backend).